t/TestWasm.pm

```diff
@@ -108,9 +108,13 @@ add_block_preprocessor(sub {
 
     my $main_config = $block->main_config || '';
 
-    $block->set_value("main_config",
-                      "env WASMTIME_BACKTRACE_DETAILS=1;\n"
```

Comment:
It just struck me this is still set even though we have now added the `backtraces` directive... It should have probably been removed at the time.